src/globalStyles.js

```diff
@@ -0,0 +1,16 @@
+// globalStyles.js
```

Comment:
Maybe you could add more detail to this file, or just clear it.

Reply:
This is the file that defines global style for the entire project, so there won't be many details in it. (but might slightly increase as the project was written)